colab/colab_utils.py

```diff
@@ -0,0 +1,1048 @@
+#@title Please run this cell first!
+
+import base64
+import collections
+import datetime
+import io
+import json
+import os
+import re
+import subprocess
+import time
+
+import ee
+import folium
+from folium.plugins import HeatMap
+import ipyplot
+import IPython.display
```

Comment:
This seems redundant with the "from IPython.display ..." statements below.

Reply:
it will be removed, thank you